.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+        required: true
+        type: string
+
+jobs:
+  tag_js_client:
+    name: JS client / Tag release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)